tests/functional/test_libraries.py

```diff
@@ -244,21 +244,28 @@ def test_PyQt5_uic(tmpdir, pyi_builder, data_dir):
     pyi_builder.test_script('pyi_lib_PyQt5-uic.py')
 
 
-@xfail(is_darwin, reason='Please help debug this. See issue #3233.')
+#@xfail(is_darwin, reason='Please help debug this. See issue #3233.')
+#@skipif_notosx
```

Comment:
Why should this test be restricted to OS X only? Just because you added arguments?